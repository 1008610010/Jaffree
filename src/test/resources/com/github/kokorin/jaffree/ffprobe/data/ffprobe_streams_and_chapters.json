{
    "streams": [
        {
            "index": 0,
            "codec_name": "hevc",
            "codec_long_name": "H.265 / HEVC (High Efficiency Video Coding)",
            "profile": "Main",
            "codec_type": "video",
            "codec_time_base": "1001/24000",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "width": 1920,
            "height": 1072,
            "coded_width": 1920,
            "coded_height": 1088,
            "has_b_frames": 0,
            "sample_aspect_ratio": "1:1",
            "display_aspect_ratio": "120:67",
            "pix_fmt": "yuv420p",
            "level": 150,
            "color_range": "tv",
            "refs": 1,
            "r_frame_rate": "24000/1001",
            "avg_frame_rate": "24000/1001",
            "time_base": "1/1000",
            "start_pts": 0,
            "start_time": "0.000000",
            "disposition": {
                "default": 1,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0
            },
            "tags": {
                "BPS": "1372729",
                "BPS-eng": "1372729",
                "DURATION": "00:22:01.445000000",
                "DURATION-eng": "00:22:01.445000000",
                "NUMBER_OF_FRAMES": "31683",
                "NUMBER_OF_FRAMES-eng": "31683",
                "NUMBER_OF_BYTES": "226748387",
                "NUMBER_OF_BYTES-eng": "226748387",
                "_STATISTICS_WRITING_APP": "mkvmerge v8.2.0 ('World of Adventure') 64bit",
                "_STATISTICS_WRITING_APP-eng": "mkvmerge v8.2.0 ('World of Adventure') 64bit",
                "_STATISTICS_WRITING_DATE_UTC": "2016-06-08 01:20:15",
                "_STATISTICS_WRITING_DATE_UTC-eng": "2016-06-08 01:20:15",
                "_STATISTICS_TAGS": "BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES",
                "_STATISTICS_TAGS-eng": "BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES"
            }
        },
        {
            "index": 1,
            "codec_name": "aac",
            "codec_long_name": "AAC (Advanced Audio Coding)",
            "profile": "LC",
            "codec_type": "audio",
            "codec_time_base": "1/48000",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "sample_fmt": "fltp",
            "sample_rate": "48000",
            "channels": 6,
            "channel_layout": "5.1",
            "bits_per_sample": 0,
            "r_frame_rate": "0/0",
            "avg_frame_rate": "0/0",
            "time_base": "1/1000",
            "start_pts": 20,
            "start_time": "0.020000",
            "disposition": {
                "default": 1,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0
            },
            "tags": {
                "language": "eng",
                "title": "English 5.1 Surround",
                "BPS": "253035",
                "BPS-eng": "253035",
                "DURATION": "00:22:01.429000000",
                "DURATION-eng": "00:22:01.429000000",
                "NUMBER_OF_FRAMES": "61942",
                "NUMBER_OF_FRAMES-eng": "61942",
                "NUMBER_OF_BYTES": "41796111",
                "NUMBER_OF_BYTES-eng": "41796111",
                "_STATISTICS_WRITING_APP": "mkvmerge v8.2.0 ('World of Adventure') 64bit",
                "_STATISTICS_WRITING_APP-eng": "mkvmerge v8.2.0 ('World of Adventure') 64bit",
                "_STATISTICS_WRITING_DATE_UTC": "2016-06-08 01:20:15",
                "_STATISTICS_WRITING_DATE_UTC-eng": "2016-06-08 01:20:15",
                "_STATISTICS_TAGS": "BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES",
                "_STATISTICS_TAGS-eng": "BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES"
            }
        },
        {
            "index": 2,
            "codec_name": "aac",
            "codec_long_name": "AAC (Advanced Audio Coding)",
            "profile": "LC",
            "codec_type": "audio",
            "codec_time_base": "1/48000",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "sample_fmt": "fltp",
            "sample_rate": "48000",
            "channels": 2,
            "channel_layout": "stereo",
            "bits_per_sample": 0,
            "r_frame_rate": "0/0",
            "avg_frame_rate": "0/0",
            "time_base": "1/1000",
            "start_pts": 20,
            "start_time": "0.020000",
            "disposition": {
                "default": 0,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0
            },
            "tags": {
                "language": "eng",
                "title": "English Commentary",
                "BPS": "76005",
                "BPS-eng": "76005",
                "DURATION": "00:22:01.472000000",
                "DURATION-eng": "00:22:01.472000000",
                "NUMBER_OF_FRAMES": "61944",
                "NUMBER_OF_FRAMES-eng": "61944",
                "NUMBER_OF_BYTES": "12554857",
                "NUMBER_OF_BYTES-eng": "12554857",
                "_STATISTICS_WRITING_APP": "mkvmerge v8.2.0 ('World of Adventure') 64bit",
                "_STATISTICS_WRITING_APP-eng": "mkvmerge v8.2.0 ('World of Adventure') 64bit",
                "_STATISTICS_WRITING_DATE_UTC": "2016-06-08 01:20:15",
                "_STATISTICS_WRITING_DATE_UTC-eng": "2016-06-08 01:20:15",
                "_STATISTICS_TAGS": "BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES",
                "_STATISTICS_TAGS-eng": "BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES"
            }
        },
        {
            "index": 3,
            "codec_name": "ass",
            "codec_long_name": "ASS (Advanced SSA) subtitle",
            "codec_type": "subtitle",
            "codec_time_base": "0/1",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "r_frame_rate": "0/0",
            "avg_frame_rate": "0/0",
            "time_base": "1/1000",
            "start_pts": 2370,
            "start_time": "2.370000",
            "disposition": {
                "default": 0,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0
            },
            "tags": {
                "language": "eng",
                "title": "English (Regular Style)",
                "BPS": "231",
                "BPS-eng": "231",
                "DURATION": "00:21:55.110000000",
                "DURATION-eng": "00:21:55.110000000",
                "NUMBER_OF_FRAMES": "719",
                "NUMBER_OF_FRAMES-eng": "719",
                "NUMBER_OF_BYTES": "37987",
                "NUMBER_OF_BYTES-eng": "37987",
                "_STATISTICS_WRITING_APP": "mkvmerge v8.2.0 ('World of Adventure') 64bit",
                "_STATISTICS_WRITING_APP-eng": "mkvmerge v8.2.0 ('World of Adventure') 64bit",
                "_STATISTICS_WRITING_DATE_UTC": "2016-06-08 01:20:15",
                "_STATISTICS_WRITING_DATE_UTC-eng": "2016-06-08 01:20:15",
                "_STATISTICS_TAGS": "BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES",
                "_STATISTICS_TAGS-eng": "BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES"
            }
        },
        {
            "index": 4,
            "codec_name": "ass",
            "codec_long_name": "ASS (Advanced SSA) subtitle",
            "codec_type": "subtitle",
            "codec_time_base": "0/1",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "r_frame_rate": "0/0",
            "avg_frame_rate": "0/0",
            "time_base": "1/1000",
            "start_pts": 2370,
            "start_time": "2.370000",
            "disposition": {
                "default": 0,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0
            },
            "tags": {
                "language": "eng",
                "title": "English (Rick & Morty Style)",
                "BPS": "231",
                "BPS-eng": "231",
                "DURATION": "00:21:55.110000000",
                "DURATION-eng": "00:21:55.110000000",
                "NUMBER_OF_FRAMES": "719",
                "NUMBER_OF_FRAMES-eng": "719",
                "NUMBER_OF_BYTES": "37994",
                "NUMBER_OF_BYTES-eng": "37994",
                "_STATISTICS_WRITING_APP": "mkvmerge v8.2.0 ('World of Adventure') 64bit",
                "_STATISTICS_WRITING_APP-eng": "mkvmerge v8.2.0 ('World of Adventure') 64bit",
                "_STATISTICS_WRITING_DATE_UTC": "2016-06-08 01:20:15",
                "_STATISTICS_WRITING_DATE_UTC-eng": "2016-06-08 01:20:15",
                "_STATISTICS_TAGS": "BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES",
                "_STATISTICS_TAGS-eng": "BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES"
            }
        },
        {
            "index": 5,
            "codec_name": "hdmv_pgs_subtitle",
            "codec_long_name": "HDMV Presentation Graphic Stream subtitles",
            "codec_type": "subtitle",
            "codec_time_base": "0/1",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "width": 1920,
            "height": 1080,
            "r_frame_rate": "0/0",
            "avg_frame_rate": "0/0",
            "time_base": "1/1000",
            "start_pts": 2377,
            "start_time": "2.377000",
            "disposition": {
                "default": 0,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0
            },
            "tags": {
                "language": "eng",
                "title": "English (BD Source)",
                "BPS": "30982",
                "BPS-eng": "30982",
                "DURATION": "00:21:55.106000000",
                "DURATION-eng": "00:21:55.106000000",
                "NUMBER_OF_FRAMES": "1438",
                "NUMBER_OF_FRAMES-eng": "1438",
                "NUMBER_OF_BYTES": "5093177",
                "NUMBER_OF_BYTES-eng": "5093177",
                "_STATISTICS_WRITING_APP": "mkvmerge v8.2.0 ('World of Adventure') 64bit",
                "_STATISTICS_WRITING_APP-eng": "mkvmerge v8.2.0 ('World of Adventure') 64bit",
                "_STATISTICS_WRITING_DATE_UTC": "2016-06-08 01:20:15",
                "_STATISTICS_WRITING_DATE_UTC-eng": "2016-06-08 01:20:15",
                "_STATISTICS_TAGS": "BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES",
                "_STATISTICS_TAGS-eng": "BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES"
            }
        },
        {
            "index": 6,
            "codec_name": "ttf",
            "codec_long_name": "TrueType font",
            "codec_type": "attachment",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "r_frame_rate": "0/0",
            "avg_frame_rate": "0/0",
            "time_base": "1/90000",
            "start_pts": 0,
            "start_time": "0.000000",
            "duration_ts": 118934280,
            "duration": "1321.492000",
            "disposition": {
                "default": 0,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0
            },
            "tags": {
                "filename": "OpenSans-Semibold.ttf",
                "mimetype": "application/x-truetype-font"
            }
        },
        {
            "index": 7,
            "codec_name": "ttf",
            "codec_long_name": "TrueType font",
            "codec_type": "attachment",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "r_frame_rate": "0/0",
            "avg_frame_rate": "0/0",
            "time_base": "1/90000",
            "start_pts": 0,
            "start_time": "0.000000",
            "duration_ts": 118934280,
            "duration": "1321.492000",
            "disposition": {
                "default": 0,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0
            },
            "tags": {
                "filename": "rick_and_morty_by_shittychan-d9h2vl6.ttf",
                "mimetype": "application/x-truetype-font"
            }
        }
    ],
    "chapters": [
        {
            "id": 950069591,
            "time_base": "1/1000000000",
            "start": 0,
            "start_time": "0.000000",
            "end": 127961166666,
            "end_time": "127.961167",
            "tags": {
                "title": "Chapter 01"
            }
        },
        {
            "id": 793586113,
            "time_base": "1/1000000000",
            "start": 127961166666,
            "start_time": "127.961167",
            "end": 159826333333,
            "end_time": "159.826333",
            "tags": {
                "title": "Chapter 02"
            }
        },
        {
            "id": 1666386186,
            "time_base": "1/1000000000",
            "start": 159826333333,
            "start_time": "159.826333",
            "end": 515348166666,
            "end_time": "515.348167",
            "tags": {
                "title": "Chapter 03"
            }
        },
        {
            "id": 1732647716,
            "time_base": "1/1000000000",
            "start": 515348166666,
            "start_time": "515.348167",
            "end": 1295294000000,
            "end_time": "1295.294000",
            "tags": {
                "title": "Chapter 04"
            }
        },
        {
            "id": -1788965165,
            "time_base": "1/1000000000",
            "start": 1295294000000,
            "start_time": "1295.294000",
            "end": 1321472000000,
            "end_time": "1321.472000",
            "tags": {
                "title": "Chapter 05"
            }
        }
    ]
}
