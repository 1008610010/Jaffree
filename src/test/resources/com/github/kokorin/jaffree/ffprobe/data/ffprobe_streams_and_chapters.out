[STREAM]
index=0
codec_name=hevc
codec_long_name=H.265 / HEVC (High Efficiency Video Coding)
profile=Main
codec_type=video
codec_time_base=1001/24000
codec_tag_string=[0][0][0][0]
codec_tag=0x0000
width=1920
height=1072
coded_width=1920
coded_height=1088
has_b_frames=0
sample_aspect_ratio=1:1
display_aspect_ratio=120:67
pix_fmt=yuv420p
level=150
color_range=tv
color_space=unknown
color_transfer=unknown
color_primaries=unknown
chroma_location=unspecified
field_order=unknown
timecode=N/A
refs=1
id=N/A
r_frame_rate=24000/1001
avg_frame_rate=24000/1001
time_base=1/1000
start_pts=0
start_time=0.000000
duration_ts=N/A
duration=N/A
bit_rate=N/A
max_bit_rate=N/A
bits_per_raw_sample=N/A
nb_frames=N/A
nb_read_frames=N/A
nb_read_packets=N/A
DISPOSITION:default=1
DISPOSITION:dub=0
DISPOSITION:original=0
DISPOSITION:comment=0
DISPOSITION:lyrics=0
DISPOSITION:karaoke=0
DISPOSITION:forced=0
DISPOSITION:hearing_impaired=0
DISPOSITION:visual_impaired=0
DISPOSITION:clean_effects=0
DISPOSITION:attached_pic=0
DISPOSITION:timed_thumbnails=0
TAG:BPS=1372729
TAG:BPS-eng=1372729
TAG:DURATION=00:22:01.445000000
TAG:DURATION-eng=00:22:01.445000000
TAG:NUMBER_OF_FRAMES=31683
TAG:NUMBER_OF_FRAMES-eng=31683
TAG:NUMBER_OF_BYTES=226748387
TAG:NUMBER_OF_BYTES-eng=226748387
TAG:_STATISTICS_WRITING_APP=mkvmerge v8.2.0 ('World of Adventure') 64bit
TAG:_STATISTICS_WRITING_APP-eng=mkvmerge v8.2.0 ('World of Adventure') 64bit
TAG:_STATISTICS_WRITING_DATE_UTC=2016-06-08 01:20:15
TAG:_STATISTICS_WRITING_DATE_UTC-eng=2016-06-08 01:20:15
TAG:_STATISTICS_TAGS=BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
TAG:_STATISTICS_TAGS-eng=BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
[/STREAM]
[STREAM]
index=1
codec_name=aac
codec_long_name=AAC (Advanced Audio Coding)
profile=LC
codec_type=audio
codec_time_base=1/48000
codec_tag_string=[0][0][0][0]
codec_tag=0x0000
sample_fmt=fltp
sample_rate=48000
channels=6
channel_layout=5.1
bits_per_sample=0
id=N/A
r_frame_rate=0/0
avg_frame_rate=0/0
time_base=1/1000
start_pts=20
start_time=0.020000
duration_ts=N/A
duration=N/A
bit_rate=N/A
max_bit_rate=N/A
bits_per_raw_sample=N/A
nb_frames=N/A
nb_read_frames=N/A
nb_read_packets=N/A
DISPOSITION:default=1
DISPOSITION:dub=0
DISPOSITION:original=0
DISPOSITION:comment=0
DISPOSITION:lyrics=0
DISPOSITION:karaoke=0
DISPOSITION:forced=0
DISPOSITION:hearing_impaired=0
DISPOSITION:visual_impaired=0
DISPOSITION:clean_effects=0
DISPOSITION:attached_pic=0
DISPOSITION:timed_thumbnails=0
TAG:language=eng
TAG:title=English 5.1 Surround
TAG:BPS=253035
TAG:BPS-eng=253035
TAG:DURATION=00:22:01.429000000
TAG:DURATION-eng=00:22:01.429000000
TAG:NUMBER_OF_FRAMES=61942
TAG:NUMBER_OF_FRAMES-eng=61942
TAG:NUMBER_OF_BYTES=41796111
TAG:NUMBER_OF_BYTES-eng=41796111
TAG:_STATISTICS_WRITING_APP=mkvmerge v8.2.0 ('World of Adventure') 64bit
TAG:_STATISTICS_WRITING_APP-eng=mkvmerge v8.2.0 ('World of Adventure') 64bit
TAG:_STATISTICS_WRITING_DATE_UTC=2016-06-08 01:20:15
TAG:_STATISTICS_WRITING_DATE_UTC-eng=2016-06-08 01:20:15
TAG:_STATISTICS_TAGS=BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
TAG:_STATISTICS_TAGS-eng=BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
[/STREAM]
[STREAM]
index=2
codec_name=aac
codec_long_name=AAC (Advanced Audio Coding)
profile=LC
codec_type=audio
codec_time_base=1/48000
codec_tag_string=[0][0][0][0]
codec_tag=0x0000
sample_fmt=fltp
sample_rate=48000
channels=2
channel_layout=stereo
bits_per_sample=0
id=N/A
r_frame_rate=0/0
avg_frame_rate=0/0
time_base=1/1000
start_pts=20
start_time=0.020000
duration_ts=N/A
duration=N/A
bit_rate=N/A
max_bit_rate=N/A
bits_per_raw_sample=N/A
nb_frames=N/A
nb_read_frames=N/A
nb_read_packets=N/A
DISPOSITION:default=0
DISPOSITION:dub=0
DISPOSITION:original=0
DISPOSITION:comment=0
DISPOSITION:lyrics=0
DISPOSITION:karaoke=0
DISPOSITION:forced=0
DISPOSITION:hearing_impaired=0
DISPOSITION:visual_impaired=0
DISPOSITION:clean_effects=0
DISPOSITION:attached_pic=0
DISPOSITION:timed_thumbnails=0
TAG:language=eng
TAG:title=English Commentary
TAG:BPS=76005
TAG:BPS-eng=76005
TAG:DURATION=00:22:01.472000000
TAG:DURATION-eng=00:22:01.472000000
TAG:NUMBER_OF_FRAMES=61944
TAG:NUMBER_OF_FRAMES-eng=61944
TAG:NUMBER_OF_BYTES=12554857
TAG:NUMBER_OF_BYTES-eng=12554857
TAG:_STATISTICS_WRITING_APP=mkvmerge v8.2.0 ('World of Adventure') 64bit
TAG:_STATISTICS_WRITING_APP-eng=mkvmerge v8.2.0 ('World of Adventure') 64bit
TAG:_STATISTICS_WRITING_DATE_UTC=2016-06-08 01:20:15
TAG:_STATISTICS_WRITING_DATE_UTC-eng=2016-06-08 01:20:15
TAG:_STATISTICS_TAGS=BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
TAG:_STATISTICS_TAGS-eng=BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
[/STREAM]
[STREAM]
index=3
codec_name=ass
codec_long_name=ASS (Advanced SSA) subtitle
profile=unknown
codec_type=subtitle
codec_time_base=0/1
codec_tag_string=[0][0][0][0]
codec_tag=0x0000
width=N/A
height=N/A
id=N/A
r_frame_rate=0/0
avg_frame_rate=0/0
time_base=1/1000
start_pts=2370
start_time=2.370000
duration_ts=N/A
duration=N/A
bit_rate=N/A
max_bit_rate=N/A
bits_per_raw_sample=N/A
nb_frames=N/A
nb_read_frames=N/A
nb_read_packets=N/A
DISPOSITION:default=0
DISPOSITION:dub=0
DISPOSITION:original=0
DISPOSITION:comment=0
DISPOSITION:lyrics=0
DISPOSITION:karaoke=0
DISPOSITION:forced=0
DISPOSITION:hearing_impaired=0
DISPOSITION:visual_impaired=0
DISPOSITION:clean_effects=0
DISPOSITION:attached_pic=0
DISPOSITION:timed_thumbnails=0
TAG:language=eng
TAG:title=English (Regular Style)
TAG:BPS=231
TAG:BPS-eng=231
TAG:DURATION=00:21:55.110000000
TAG:DURATION-eng=00:21:55.110000000
TAG:NUMBER_OF_FRAMES=719
TAG:NUMBER_OF_FRAMES-eng=719
TAG:NUMBER_OF_BYTES=37987
TAG:NUMBER_OF_BYTES-eng=37987
TAG:_STATISTICS_WRITING_APP=mkvmerge v8.2.0 ('World of Adventure') 64bit
TAG:_STATISTICS_WRITING_APP-eng=mkvmerge v8.2.0 ('World of Adventure') 64bit
TAG:_STATISTICS_WRITING_DATE_UTC=2016-06-08 01:20:15
TAG:_STATISTICS_WRITING_DATE_UTC-eng=2016-06-08 01:20:15
TAG:_STATISTICS_TAGS=BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
TAG:_STATISTICS_TAGS-eng=BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
[/STREAM]
[STREAM]
index=4
codec_name=ass
codec_long_name=ASS (Advanced SSA) subtitle
profile=unknown
codec_type=subtitle
codec_time_base=0/1
codec_tag_string=[0][0][0][0]
codec_tag=0x0000
width=N/A
height=N/A
id=N/A
r_frame_rate=0/0
avg_frame_rate=0/0
time_base=1/1000
start_pts=2370
start_time=2.370000
duration_ts=N/A
duration=N/A
bit_rate=N/A
max_bit_rate=N/A
bits_per_raw_sample=N/A
nb_frames=N/A
nb_read_frames=N/A
nb_read_packets=N/A
DISPOSITION:default=0
DISPOSITION:dub=0
DISPOSITION:original=0
DISPOSITION:comment=0
DISPOSITION:lyrics=0
DISPOSITION:karaoke=0
DISPOSITION:forced=0
DISPOSITION:hearing_impaired=0
DISPOSITION:visual_impaired=0
DISPOSITION:clean_effects=0
DISPOSITION:attached_pic=0
DISPOSITION:timed_thumbnails=0
TAG:language=eng
TAG:title=English (Rick & Morty Style)
TAG:BPS=231
TAG:BPS-eng=231
TAG:DURATION=00:21:55.110000000
TAG:DURATION-eng=00:21:55.110000000
TAG:NUMBER_OF_FRAMES=719
TAG:NUMBER_OF_FRAMES-eng=719
TAG:NUMBER_OF_BYTES=37994
TAG:NUMBER_OF_BYTES-eng=37994
TAG:_STATISTICS_WRITING_APP=mkvmerge v8.2.0 ('World of Adventure') 64bit
TAG:_STATISTICS_WRITING_APP-eng=mkvmerge v8.2.0 ('World of Adventure') 64bit
TAG:_STATISTICS_WRITING_DATE_UTC=2016-06-08 01:20:15
TAG:_STATISTICS_WRITING_DATE_UTC-eng=2016-06-08 01:20:15
TAG:_STATISTICS_TAGS=BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
TAG:_STATISTICS_TAGS-eng=BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
[/STREAM]
[STREAM]
index=5
codec_name=hdmv_pgs_subtitle
codec_long_name=HDMV Presentation Graphic Stream subtitles
profile=unknown
codec_type=subtitle
codec_time_base=0/1
codec_tag_string=[0][0][0][0]
codec_tag=0x0000
width=1920
height=1080
id=N/A
r_frame_rate=0/0
avg_frame_rate=0/0
time_base=1/1000
start_pts=2377
start_time=2.377000
duration_ts=N/A
duration=N/A
bit_rate=N/A
max_bit_rate=N/A
bits_per_raw_sample=N/A
nb_frames=N/A
nb_read_frames=N/A
nb_read_packets=N/A
DISPOSITION:default=0
DISPOSITION:dub=0
DISPOSITION:original=0
DISPOSITION:comment=0
DISPOSITION:lyrics=0
DISPOSITION:karaoke=0
DISPOSITION:forced=0
DISPOSITION:hearing_impaired=0
DISPOSITION:visual_impaired=0
DISPOSITION:clean_effects=0
DISPOSITION:attached_pic=0
DISPOSITION:timed_thumbnails=0
TAG:language=eng
TAG:title=English (BD Source)
TAG:BPS=30982
TAG:BPS-eng=30982
TAG:DURATION=00:21:55.106000000
TAG:DURATION-eng=00:21:55.106000000
TAG:NUMBER_OF_FRAMES=1438
TAG:NUMBER_OF_FRAMES-eng=1438
TAG:NUMBER_OF_BYTES=5093177
TAG:NUMBER_OF_BYTES-eng=5093177
TAG:_STATISTICS_WRITING_APP=mkvmerge v8.2.0 ('World of Adventure') 64bit
TAG:_STATISTICS_WRITING_APP-eng=mkvmerge v8.2.0 ('World of Adventure') 64bit
TAG:_STATISTICS_WRITING_DATE_UTC=2016-06-08 01:20:15
TAG:_STATISTICS_WRITING_DATE_UTC-eng=2016-06-08 01:20:15
TAG:_STATISTICS_TAGS=BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
TAG:_STATISTICS_TAGS-eng=BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
[/STREAM]
[STREAM]
index=6
codec_name=ttf
codec_long_name=TrueType font
profile=unknown
codec_type=attachment
codec_tag_string=[0][0][0][0]
codec_tag=0x0000
id=N/A
r_frame_rate=0/0
avg_frame_rate=0/0
time_base=1/90000
start_pts=0
start_time=0.000000
duration_ts=118934280
duration=1321.492000
bit_rate=N/A
max_bit_rate=N/A
bits_per_raw_sample=N/A
nb_frames=N/A
nb_read_frames=N/A
nb_read_packets=N/A
DISPOSITION:default=0
DISPOSITION:dub=0
DISPOSITION:original=0
DISPOSITION:comment=0
DISPOSITION:lyrics=0
DISPOSITION:karaoke=0
DISPOSITION:forced=0
DISPOSITION:hearing_impaired=0
DISPOSITION:visual_impaired=0
DISPOSITION:clean_effects=0
DISPOSITION:attached_pic=0
DISPOSITION:timed_thumbnails=0
TAG:filename=OpenSans-Semibold.ttf
TAG:mimetype=application/x-truetype-font
[/STREAM]
[STREAM]
index=7
codec_name=ttf
codec_long_name=TrueType font
profile=unknown
codec_type=attachment
codec_tag_string=[0][0][0][0]
codec_tag=0x0000
id=N/A
r_frame_rate=0/0
avg_frame_rate=0/0
time_base=1/90000
start_pts=0
start_time=0.000000
duration_ts=118934280
duration=1321.492000
bit_rate=N/A
max_bit_rate=N/A
bits_per_raw_sample=N/A
nb_frames=N/A
nb_read_frames=N/A
nb_read_packets=N/A
DISPOSITION:default=0
DISPOSITION:dub=0
DISPOSITION:original=0
DISPOSITION:comment=0
DISPOSITION:lyrics=0
DISPOSITION:karaoke=0
DISPOSITION:forced=0
DISPOSITION:hearing_impaired=0
DISPOSITION:visual_impaired=0
DISPOSITION:clean_effects=0
DISPOSITION:attached_pic=0
DISPOSITION:timed_thumbnails=0
TAG:filename=rick_and_morty_by_shittychan-d9h2vl6.ttf
TAG:mimetype=application/x-truetype-font
[/STREAM]
[CHAPTER]
id=950069591
time_base=1/1000000000
start=0
start_time=0.000000
end=127961166666
end_time=127.961167
TAG:title=Chapter 01
[/CHAPTER]
[CHAPTER]
id=793586113
time_base=1/1000000000
start=127961166666
start_time=127.961167
end=159826333333
end_time=159.826333
TAG:title=Chapter 02
[/CHAPTER]
[CHAPTER]
id=1666386186
time_base=1/1000000000
start=159826333333
start_time=159.826333
end=515348166666
end_time=515.348167
TAG:title=Chapter 03
[/CHAPTER]
[CHAPTER]
id=1732647716
time_base=1/1000000000
start=515348166666
start_time=515.348167
end=1295294000000
end_time=1295.294000
TAG:title=Chapter 04
[/CHAPTER]
[CHAPTER]
id=-1788965165
time_base=1/1000000000
start=1295294000000
start_time=1295.294000
end=1321472000000
end_time=1321.472000
TAG:title=Chapter 05
[/CHAPTER]
